app/client/rest/base.ts

```diff
@@ -318,58 +258,22 @@ export default class ClientBase {
             });
         }
 
-        if (headers.has(ClientConstants.HEADER_X_VERSION_ID) && !headers.get('Cache-Control')) {
-            const serverVersion = headers.get(ClientConstants.HEADER_X_VERSION_ID);
-            if (serverVersion && this.serverVersion !== serverVersion) {
-                this.serverVersion = serverVersion;
-            }
-        }
-
-        if (headers.has(ClientConstants.HEADER_X_CLUSTER_ID)) {
-            const clusterId = headers.get(ClientConstants.HEADER_X_CLUSTER_ID);
-            if (clusterId && this.clusterId !== clusterId) {
-                this.clusterId = clusterId;
-            }
+        const headers = response.headers;
+        const serverVersion = headers[ClientConstants.HEADER_X_VERSION_ID] || headers[ClientConstants.HEADER_X_VERSION_ID.toLowerCase()];
+        if (serverVersion && !headers['Cache-Control'] && this.serverVersion !== serverVersion) {
+            this.serverVersion = serverVersion;
+            // EventEmitter.emit(General.SERVER_VERSION_CHANGED, serverVersion);
```

Comment:
We should be emitting here, let use the EventEmitter from RN and not the one we had on mm-redux